File: server/src/main/java/rs/ac/singidunum/novisad/server/model/korisnik/PravoPristupa.java
package rs.ac.singidunum.novisad.server.model.korisnik;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity
public class PravoPristupa {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @ManyToOne
    private Uloga uloga;
    @ManyToOne()
    private RegistrovaniKorisnik registrovaniKorisnik;

}

File: client/src/app/model/pravoPristupa.ts
import {RegistrovaniKorisnik} from "./registrovaniKorisnik";
import {Uloga} from "./uloga";
export interface PravoPristupa {
  id: number|undefined,
  uloga: Uloga|undefined,
  registrovaniKorisnik: RegistrovaniKorisnik|null
}
